src/_includes/icon-promo.css

```diff
@@ -0,0 +1,69 @@
+.root {
+  position: relative;
+  z-index: 1;
+  text-align: center;
+  --radius: 14px;
+  border-radius: var(--radius);
+  margin: calc(var(--radius) * -1) 0;
+
+  @media (min-width: 520px) {
+    --vertical-padding: 5rem;
+  }
+}
+
+.icon {
+  height: 3.5rem;
+  display: flex;
+  justify-content: center;
+
+  @media (min-width: 520px) {
+    height: 2.5rem;
+  }
+
+  & > :first-child {
+    height: 100%;
+  }
+}
+
+.title {
+  font-family: google sans, roboto, sans-serif;
+  color: var(--offblack);
+  margin: 1.4em 0 1em;
+  font-size: 1.8rem;
+  line-height: 1.3;
+}
+
+.blurb {
+  max-width: 25rem;
```

Comment:
```suggestion
  max-width: 35ch;
```

pacify scenarios that result in widows

before:
![Screen Shot 2019-10-10 at 9 12 38 AM](https://user-images.githubusercontent.com/1134620/66586938-5484cd80-eb3e-11e9-8ab4-dee6352e62ac.png)


after:
![Screen Shot 2019-10-10 at 9 12 15 AM](https://user-images.githubusercontent.com/1134620/66586945-59498180-eb3e-11e9-939e-41a6c83b1cf7.png)